terraform/modules/queue/main.tf

```diff
@@ -47,10 +47,11 @@ data "aws_iam_policy_document" "sns_send_message" {
       values   = [var.sns_topic_arn]
```

Comment:
Let's remove the above statement and count conditionals (and remove the aws_sns_topic_subscription resource below as well.). Then update [cclf-import and opt-out-import](https://github.com/search?q=repo%3ACMSgov%2Fcdap+sns_topic_arn+path%3A%2F%5Eterraform%5C%2Fservices%5C%2F%2F&type=code) to pass in a policy document as you've done in cost-anomaly.